src/qt/bitcoingui.cpp

```diff
@@ -251,28 +252,28 @@ void BitcoinGUI::createActions()
     overviewAction->setStatusTip(tr("Show general overview of wallet"));
     overviewAction->setToolTip(overviewAction->statusTip());
     overviewAction->setCheckable(true);
-    overviewAction->setShortcut(QKeySequence(Qt::ALT + Qt::Key_1));
+    overviewAction->setShortcut(QKeySequence(QString{"Alt+1"}));
```

Comment:
0fea3846d0d6ec69dadd8bf043afd3c25971d07f

nit, here and below, use `QStringLiteral`, see https://doc.qt.io/qt-5/qstring.html#more-efficient-string-construction

Reply:
Thanks! [Updated](https://github.com/bitcoin/bitcoin/pull/24624#issuecomment-1074850209).